cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py

```diff
@@ -42,6 +45,41 @@ def _is_segm_id_test(self, param, name):
         return param in PARAMS_NAMES_TO_OMIT and (
             name in NET_NAMES_TO_OMIT or name in SUBNET_NAMES_TO_OMIT)
 
+    def get_attribute_val(self, attr_list, resource_name, parameter):
+        temp = {}
+        temp[resource_name] = {}
+        temp[resource_name][parameter] = getattr(attr_list, parameter)
+        temp[resource_name]["name"] = getattr(attr_list, "name").lower()
+        return temp[resource_name][parameter]
+
+    def validate_tenant_resource_param_in_dst(self, src_list, dst_list,
+                                              resource_name='tenant',
+                                              parameter='name'):
+        if not src_list:
+            self.skipTest(
+                'Nothing to migrate - source resources list is empty')
```

Comment:
Please put more information into this message, such as `resource_name` and `parameter`, it would be much easier to troubleshoot